src/BuildScriptGenerator/Node/NodeConstants.cs

```diff
@@ -18,6 +19,15 @@ public static class NodeConstants
         public const string YarnStartCommand = "yarn run start";
         public const string YarnCommand = "yarn";
         public const string HugoCommand = "hugo";
+        public const string LernaCommand = "lerna";
+        public const string LernaVersionCommand = "echo Using Npm version: && lerna --version";
+        public const string LernaInitCommand = "lerna init";
+        public const string LernaBootstrapCommand = "lerna bootstrap";
+        public const string InstallLernaCommandNpm = "npm install --global lerna";
+        public const string InstallLernaCommandYarn = "npm install --global lerna --no-package-lock";
```

Comment:
What is the difference between `InstallLernaCommandNpm` and `InstallLernaCommandYarn`? I do not see `yarn` being used in the `Yarn` command

Reply:
Good point, its basically same, I was thinking not adding `package-lock.json` if it's yarn, so added it, but it seems not necessary since it will not create a brand new one if not exist one previously.